com.adsbynimbus.nimbus/Runtime/Plugins/iOS/NimbusManager.swift

```diff
@@ -29,20 +29,31 @@ import GoogleMobileAds
 import NimbusRequestKit
 import NimbusSDK
 #endif
+#if NIMBUS_ENABLE_MINTEGRAL
+import MTGSDK
+import MTGSDKBidding
+import NimbusSDK
+#endif
 
 @objc public class NimbusManager: NSObject {
     
     private static var managerDictionary: [Int: NimbusManager] = [:]
     
     private let adUnitInstanceId: Int
+    
     private var nimbusAdManager: NimbusAdManager?
     private var adController: AdController?
 
-    
     #if NIMBUS_ENABLE_APS
     private static var apsRequestHelper: NimbusAPSRequestHelper?
     #endif
     
+    #if NIMBUS_ENABLE_MINTEGRAL
+    private static var mintegralInterceptor: NimbusRequestInterceptor?
+    private static var mintegralAdUnitId: String = ""
```

Comment:
The use of static variables here seems likely to introduce a race condition. 

For example, if request a rewarded ad and an interstitial ad at the same time and then decide to render the rewarded ad, which which placement id is it going to use?